### ISSUE TITLE
HSTS

### ISSUE THREAD

**Issue by annevk** [unlicense/unlicense.org#53]
It would be nice if the HTTP version of unlicense.org redirected to HTTPS and all the requirements in https://hstspreload.org/ were met so it can be submitted.

**Comment by Hexstream** [comment]
Yes, also there are mixed content errors on the [HTTPS version](https://unlicense.org/) of the site (press `F12` and check the console).

For what it's worth, my strategy these days is to put **everything** behind CDNs, which then handle all the SSL/TLS stuff with very minimal setup. All my sites are HTTPS-only.

I currently use [CloudFront](https://aws.amazon.com/cloudfront/) (which is dirt-cheap) for most things and [Cloudflare](https://www.cloudflare.com/) for other things, which is free if you're ok with the limitations of their free SSL/TLS (else it's 5$ or 10$ a month for a "dedicated" certificate) and is quite a bit easier to setup. (I would move everything to Cloudflare if they had "proper" free certificates.)

Hope this helps.

**Comment by johnjago** [comment]
@Hexstream I believe the mixed content errors can be fixed by simply adding the "s" to those links in the source code of the site. Both offer HTTPS versions. Although the Yahoo one appears to be down.

For a small site like this, editing the server configuration to always redirect to HTTPS should not take more than an hour.

**Comment by bendiken** [comment]
Thanks. We will come back to this after the ongoing site revamp is done.

**Comment by Hexstream** [comment]
Looking forward to the site revamp!

(btw, regarding my previous comment, now all my sites are 100% on DigitalOcean through Cloudflare.)
